src/neuro_flow/batch_executor.py

```diff
@@ -1301,8 +1302,9 @@ async def _process_local(self, full_id: FullID) -> None:
             status=TaskStatus.PENDING,
             raw_id=None,
         )
+        posix = False if sys.platform == "win32" else True
         subprocess = await asyncio.create_subprocess_exec(
-            *shlex.split(local.cmd),
+            *shlex.split(local.cmd, posix=posix),
```

Comment:
required to fix [this](https://github.com/neuro-inc/neuro-flow/actions/runs/4651153542/jobs/8230729853#step:7:282) one.
tl; dr: `${{ flow.workspace }}` in `Local` actions on Windows machines was rendered without any backslashes, e.g. `D:\path\to\folder` was rendered as `Dpathtofolder`.
I cannot say why it was like this previously, since this error each time in e2e tests on all Py versions on Windows.